sdk/python/feast/offline_server.py

```diff
@@ -20,6 +24,7 @@ def __init__(self, store: FeatureStore, location: str, **kwargs):
         # A dictionary of configured flights, e.g. API calls received and not yet served
         self.flights: Dict[str, Any] = {}
         self.store = store
+        self.offline_store = get_offline_store_from_config(store.config.offline_store)
```

Comment:
please use the same `self.offline_store` reference for the `get_historical_features` method